sql/queries/swim-trackpoints.sql

```diff
@@ -1,7 +1,7 @@
 -- fetch-swim-trackpoints.sql -- fetch trackpoints for a swim activtiy
 --
 -- This file is part of ActivityLog2 -- https://github.com/alex-hhh/ActivityLog2
--- Copyright (c) 2018 Alex Hars�nyi <AlexHarsanyi@gmail.com>
+-- Copyright (c) 2018 Alex Hars�nyi <AlexHarsanyi@gmail.com>
```

Comment:
Can you revert this change -- I think you need to set the file save to UTF-8 in your editor?